acquia_lift.admin.inc

```diff
@@ -936,7 +895,7 @@ function acquia_lift_target_audience_save($label, $agent_name, $contexts, $strat
     $option_set->targeting[$machine_name] = array();
   }
   $option_set->targeting[$machine_name]['label'] = $label;
-  $option_set->targeting[$machine_name]['weight'] = count($option_set->targeting);
+  $option_set->targeting[$machine_name]['weight'] = $weight;
```

Comment:
I'm wondering what we should do if there's already an audience with that weight - should we keep incrementing / decrementing it until we get a weight that no other audience uses?